src/Microsoft.DotNet.Interactive.Tests/InputsWithinMagicCommandsTests.cs

```diff
@@ -237,6 +237,46 @@ public async Task Additional_properties_of_input_request_are_set_by_input_proper
         requestInput.InputTypeHint.Should().Be("file");
     }
 
+    [Theory]
+    [MemberData(nameof(LanguageServiceCommands))]
+    public async Task Language_service_commands_do_not_trigger_input_requests(KernelCommand command)
```

Comment:
If this test fails - will this display the name of the command that is failing? It would be nice if we could do something so that it is easier to know the command (and code) for the failure case

Reply:
Yes, test output shows the parameters for the failing case.

Reply:
Ah was asking if we can tell by looking at the view in the test runner itself without clicking to inspect the output. But not a huge issue either way.

Reply:
Yes, most test runners support this.